ktor-network/posix/src/io/ktor/network/util/Pools.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.util
+
+import io.ktor.network.sockets.*
+import io.ktor.utils.io.core.*
+import io.ktor.utils.io.pool.*
+
+@ThreadLocal
+public val DefaultDatagramByteBufferPool: ObjectPool<IoBuffer> =
```

Comment:
Could you tell me why this pool is public?

Reply:
Sorry, it should have the `@InternalAPI` annotation, I fixed that in latest commit. The reason I made this public is because the JVM version of this val is public as well, so I tried to keep it similar:
https://github.com/ktorio/ktor/blob/815f247050cd83e39dfd0a7454e795b206907977/ktor-network/jvm/src/io/ktor/network/util/Pools.kt#L45-L49
If you still would like me to change it to internal, let me know.